spec/humanize_spec.rb

```diff
@@ -19,12 +17,48 @@
 
     it 'uses default locale' do
       Humanize.config.default_locale = :fr
-      expect(42.humanize).to eql("quarante-deux".gsub(/ +/, ' '))
+      expect(42.humanize).to eql('quarante-deux')
     end
 
     it 'uses locale passed as argument if given' do
       Humanize.config.default_locale = :en
-      expect(42.humanize(:locale => :fr)).to eql("quarante-deux".gsub(/ +/, ' '))
+      expect(42.humanize(:locale => :fr)).to eql('quarante-deux')
+    end
+
+    describe 'french specific rules' do
+
+      it 'one thousand and one equals "mille un"' do
+        expect(1002.humanize(:locale => :fr)).to eql('mille deux')
+      end
+
+      it 'two thousand and one equals "deux mille un"' do
```

Comment:
I did not feel it was necessary, having tested _some_ thousand with 2001.
But here it is !